examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def op_shallowWaterWave(etasave, eta, M, N, h, D, g, alpha):
</pre><p>Do we use camel case for functions in Devito? I think <code>op_shallowwaterwave</code> would be more consistent with the style of the rest of the codebase. Double check this though </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I think we use </p><pre class="ql-syntax" spellcheck="false">def ForwardOperator()

</pre>

Reply:
<p>Hm, should I remove the underline? I'm not sure, but the camel case makes sense.</p>